### PR TITLE
add simdjson

### PR DESCRIPTION
**Pull request by paul-reilly**
This passes relevant CI on my fork. Please let me know if anything could be improved.

### CODE REVIEW

File: packages/s/simdjson/xmake.lua [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        configs.logging = package:config("logging")
+
+        os.cp(path.join(package:scriptdir(), "port", "xmake.lua"), "xmake.lua")
+        import("package.tools.xmake").install(package, configs)
+    end)
+
+    on_test(function(package)
+        assert(package:check_cxxsnippets({ test = [[
+            
+                simdjson::dom::parser parser;
```
Comment (waruqi) [inline packages/s/simdjson/xmake.lua:30]: please remove empty lines

File: packages/s/simdjson/port/xmake.lua [hunk 2]
```diff
@@ -0,0 +1,29 @@
+add_rules("mode.release")
```
Comment (waruqi) [inline packages/s/simdjson/port/xmake.lua:1]: add mode.debug rule

File: packages/s/simdjson/port/xmake.lua [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    add_defines("SIMDJSON_VERBOSE_LOGGING")
+
+
+target("simdjson")   
+    set_languages("c++17")
+    set_kind("$(kind)")
+    add_options("threads", "noexceptions", "logging")
+    add_files("singleheader/simdjson.cpp")
+    add_headerfiles("singleheader/simdjson.h")
+    add_defines("NDEBUG")
```
Comment (waruqi) [inline packages/s/simdjson/port/xmake.lua:28]: mode.release rule will add NDEBUG definition

Comment (paul-reilly) [reply packages/s/simdjson/port/xmake.lua]: Ah, okay thanks.

File: packages/s/simdjson/port/xmake.lua (hunk 2 above)
Comment (paul-reilly) [reply packages/s/simdjson/port/xmake.lua]: Done.

File: packages/s/simdjson/xmake.lua (hunk 1 above)
Comment (paul-reilly) [reply packages/s/simdjson/xmake.lua:29]: Done, cheers.

**Comment by waruqi** [comment]
thanks.

**Pull request merged**